tests/lax_numpy_test.py

```diff
@@ -1329,7 +1357,11 @@ def testUnique(self, shape, dtype, return_index, return_inverse, return_counts):
     jnp_fun = lambda x: jnp.unique(x, return_index, return_inverse, return_counts)
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker)
 
-  def testIssue1233(self):
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_fixed_size={}".format(fixed_size),
+      "fixed_size": fixed_size}
+      for fixed_size in [True, False]))
+  def testIssue1233(self, fixed_size):
```

Comment:
Not related to your PR, but I'd rename this to something like `testNonScalarRepeat` or something more descriptive.